### PR TITLE
docs: add Debian and Ubuntu packaging and install

### PR DESCRIPTION
**Pull request by samuelkarp**
docs: add Debian and Ubuntu packaging and install

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,37 @@ Once you have installed the credential helper, see the
 [Configuration section](#Configuration) for instructions on how to configure
 Docker to work with the helper.
 
+### Debian Buster (testing) and Debian Sid (unstable)
```
Comment (nmeyerhans) [inline README.md]: I'd try to take a more general approach here. Buster won't be testing forever, and there will be a new testing at some point. This text is thus subject to bitrot. Might be worth instead describing that this package will be included in buster and future releases without assuming specific name mappings.

File: README.md [hunk 2]
```diff
@@ -57,6 +57,37 @@ Once you have installed the credential helper, see the
...
+```bash
+$ sudo apt update
+$ sudo apt install amazon-ecr-credential-helper
+```
+
+Once you have installed the credential helper, see the
+[Configuration section](#Configuration) for instructions on how to configure
+Docker to work with the helper.
+
+### Ubuntu 19.04 Disco Dingo
```
Comment (nmeyerhans) [inline README.md]: Similar to Debian, I'd try to pick language that'll continue to be meaningful as future releases go. We can expect the package to be installable on 19.10, but this language will be broken. Maybe "19.04 and newer" or "19.04+" or something like that.

File: README.md (hunk 1 above)
Comment (samuelkarp) [reply README.md]: My intent was just to update this when Buster becomes stable.  Do you think that's too complicated?

File: README.md (hunk 2 above)
Comment (samuelkarp) [reply README.md]: As above

File: README.md (hunk 1 above)
Comment (nmeyerhans) [reply README.md]: Yeah, seems annoying to have to make commits to this repo just to note activity in third-party projects.

Comment (samuelkarp) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (samuelkarp) [reply README.md]: Done

**Pull request merged**